### PR TITLE
Support neural-chat-7b model for chatbot

### PR DESCRIPTION
**Pull request by lvliang-intel**
## Type of Change

feature
API not changed

## Description

Support Intel neural-chat-7b model

## Expected Behavior & Potential Risk

The neural-chat-7b model can be run successfully.

## How has this PR been tested?

Local test.

## Dependency Change?

No.

### CODE REVIEW

**Review by sywangyi — CHANGES_REQUESTED** [review]
how to handle Intel/neural-chat-7b-v2, it's llama instead of mpt

**Comment by lvliang-intel** [comment]
> how to handle Intel/neural-chat-7b-v2, it's llama instead of mpt

Fixed.

**Review by sywangyi — APPROVED** [review]

File: workflows/chatbot/inference/generate.py [hunk 1]
```diff
@@ -338,14 +338,16 @@ def load_model(
...
+            or re.search("neural-chat-7b-v2", model_name, re.IGNORECASE)) else True,
     )
     if re.search("flan-t5", model_name, re.IGNORECASE):
         with smart_context_manager(use_deepspeed=use_deepspeed):
             model = AutoModelForSeq2SeqLM.from_pretrained(
                 model_name, low_cpu_mem_usage=True
             )
-    elif re.search("mpt", model_name, re.IGNORECASE):
+    elif (re.search("mpt", model_name, re.IGNORECASE)
+        or re.search("neural-chat-7b-v1", model_name, re.IGNORECASE)):
```
Comment (hshen14) [inline workflows/chatbot/inference/generate.py:350]: please use Intel/neural-chat-7b-v1-1, since v1.1 is trained with longer seq length.

Comment (lvliang-intel) [reply workflows/chatbot/inference/generate.py:350]: not an issue.

**Review by hshen14 — APPROVED** [review]

**Pull request merged**